crates/rpc/rpc/src/debug.rs

```diff
@@ -104,6 +104,15 @@ where
             return Ok(Vec::new())
         }
 
+        // Retrieve the block from the block ID
+        let block = match self.eth_api().block_with_senders(at).await? {
+            None => return Err(EthApiError::HeaderNotFound(at).into()),
+            Some(block) => block,
+        };
+
+        // Extract the parent beacon block root from the block
+        let parent_beacon_block_root = block.parent_beacon_block_root;
```

Comment:
I believe this shouldn't be required here, because we always fetch the block before we call this function.

so we can likely solve this by adding the parent beacon block root as another argument to this function